src/EnergyPlus/DXCoils.cc

```diff
@@ -8405,13 +8406,14 @@ Label50: ;
 			// Modify total heating capacity based on defrost heating capacity multiplier
 			// MaxHeatCap passed from parent object VRF Condenser and is used to limit capacity of TU's to that available from condenser
 			if ( present( MaxHeatCap ) ) {
-				TotCap = min( MaxHeatCap, TotCap * HeatingCapacityMultiplier );
+				TotCapAdj = min( MaxHeatCap, TotCap * HeatingCapacityMultiplier );
+				TotCap = min( MaxHeatCap, TotCap );
 			} else {
-				TotCap *= HeatingCapacityMultiplier;
+				TotCapAdj = TotCap * HeatingCapacityMultiplier;
 			}
```

Comment:
OK, right. This IF only applies the limit of MaxHeatCap if VRF is used. Otherwise, TotCap remains the same.